KLR/Util/Sexp.lean

```diff
@@ -47,12 +47,21 @@ export ToSexp(toSexp)
 instance : ToSexp String where
   toSexp n := atom n
 
+instance : ToSexp Bool where
+  toSexp | true => atom "#t"
```

Comment:
This is scheme-specific. Even lisp/elisp don't use it. I guess I'd prefer just true and false.